packages/viz/src/lib/sources/Source.ts

```diff
@@ -2,7 +2,11 @@
  *
  * Base Source definition. We should keep here the code shared between different sources
  */
-import { GeometryType, NumericFieldStats, CategoryFieldStats } from '../types';
+import {
+  GeometryType,
+  NumericFieldStats,
+  CategoryFieldStats
+} from '../global-interfaces';
```

Comment:
It's ok to have now that name, but I think we should probably rename that into something more meaningful in the future (maybe exposed them from `./Metadata`?)

Reply:
Yes, It'm not happy with that solution. In fact, I've changed the name multiple times

Reply:
Done!